packages/@ngtools/webpack/src/paths-plugin.ts

```diff
@@ -87,76 +81,3 @@ export function resolveWithPaths(
   request.request = moduleFilePath;
   callback(null, request);
 }
-
-export interface PathsPluginOptions {
-  nmf: NormalModuleFactory;
-  tsConfigPath: string;
-  compilerOptions?: ts.CompilerOptions;
-  compilerHost?: ts.CompilerHost;
-}
-
-export class PathsPlugin implements Tapable {
```

Comment:
What are we using in Ng5+ for TS path resolution?

Reply:
AngularCompilerPlugin uses the `resolveWithPaths` function directly: https://github.com/angular/angular-cli/blob/master/packages/%40ngtools/webpack/src/angular_compiler_plugin.ts#L654